distribution/integ-test-zip/build.gradle

```diff
@@ -35,5 +37,14 @@ publishing {
   }
 }
 
+Task integTestNetty4 = tasks.create("integTestNetty4", RestIntegTestTask) {
```

Comment:
adding that class works! I think we should use more than one node in the integTests to also use the netty transport though.

```
integTest {
    includePackaged = true
    cluster {
        setting 'http.type', 'netty4'
        setting 'transport.type', 'netty4'
        numNodes = 2
    }
}
```